src/main/java/Quinn.java

```diff
@@ -19,15 +19,29 @@ public void run() {
             System.out.print("Enter command: \t");
```

Comment:
I like the use of the empty lines within methods to differentiate between different tasks!